### PR TITLE
Restore prod

### PR DESCRIPTION
**Pull request by AlexMelov**

### CODE REVIEW

File: frontend/.eslintrc [hunk 1]
```diff
@@ -27,7 +27,13 @@
 			"extends":
 			[
 				"plugin:@angular-eslint/template/process-inline-templates"
-			]
+			],
+			"parserOptions": {
+				"project":
```
Comment (henryruhs) [inline frontend/.eslintrc:33]: Please rewrite to this - I think ** includes cypress folder aswell

```
"project":
[
	"**/tsconfig.json"
]
```

File: frontend/src/app/todo/form/form.component.spec.ts [hunk 2]
```diff
@@ -33,7 +33,11 @@ describe('ListComponent', () =>
 		service.getAll().subscribe(todos =>
 		{
 			expect(todos).not.toHaveLength(0);
-			expect(todos[0].id).toBeTruthy();
+			todos.forEach(todo =>
```
Comment (henryruhs) [inline frontend/src/app/todo/form/form.component.spec.ts]: Should not be part of this PR

File: frontend/src/app/todo/list/list.component.spec.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ describe('ListComponent', () =>
 		service = TestBed.inject(TodoService);
 	});
 
-	it('should mock the create', async() =>
+	it('should create todo', async() =>
```
Comment (henryruhs) [inline frontend/src/app/todo/list/list.component.spec.ts]: Should not be part of this PR

File: frontend/src/app/todo/todo.component.scss [hunk 4]
```diff
@@ -3,3 +3,11 @@
 	margin: 0 auto;
 	max-width: 50rem;
 }
+.progress-bar
```
Comment (henryruhs) [inline frontend/src/app/todo/todo.component.scss]: Should not be part of this PR

File: frontend/src/app/todo/todo.service.spec.ts [hunk 5]
```diff
@@ -16,13 +16,16 @@ describe('ListComponent', () =>
 		service = TestBed.inject(TodoService);
 	});
 
-	it('should mock the create', async() =>
+	it('should get all todos', async() =>
```
Comment (henryruhs) [inline frontend/src/app/todo/todo.service.spec.ts]: Should not be part of this PR

File: frontend/src/app/todo/todo.service.spec.ts [hunk 6]
```diff
@@ -16,13 +16,16 @@ describe('ListComponent', () =>
...
 	{
 		expect(service).toBeTruthy();
 
 		service.getAll().subscribe(todos =>
 		{
-			expect(todos).not.toHaveLength(0);
+			todos.forEach(todo =>
```
Comment (henryruhs) [inline frontend/src/app/todo/todo.service.spec.ts]: Should not be part of this PR

File: frontend/src/ui.module.ts [hunk 7]
```diff
@@ -2,14 +2,16 @@ import { NgModule } from '@angular/core';
...
+import { MatToolbarModule } from '@angular/material/toolbar';
 
 @NgModule(
 {
 	exports:
 	[
 		MatButtonModule,
 		MatListModule,
-		MatInputModule
+		MatInputModule,
```
Comment (henryruhs) [inline frontend/src/ui.module.ts]: Should not be part of this PR

File: frontend/tsconfig.json [hunk 8]
```diff
@@ -19,10 +19,6 @@
 		"target": "es2017",
 		"module": "es2020",
 		"skipLibCheck": true,
```
Comment (henryruhs) [inline frontend/tsconfig.json]: This might cause problems during lint!?

**Review by henryruhs — CHANGES_REQUESTED** [review]

File: frontend/.eslintrc (hunk 1 above)
Comment (AlexMelov) [reply frontend/.eslintrc:33]: **/cypress/tsconfig.json not working, lint throws errors like he don't recognize the cypress syntax

**Pull request merged**